cfgov/unprocessed/root/financial-intuition.xml

```diff
@@ -16,6 +16,51 @@
         />
         <itunes:category text="Education"></itunes:category>
         <itunes:explicit>false</itunes:explicit>
+        <item>
+            <title>Managing Your Student Loans, Part 3: Postgraduate Degree Repayment Options</title>
+            <link>https://www.consumerfinance.gov/practitioner-resources/students/financial-intuition/managing-your-student-loans-part-3/</link>
+            <itunes:episodeType>full</itunes:episodeType>
+            <itunes:episode>6</itunes:episode>
+            <description>For the third and final episode of this series, we spoke with Dr. Bridget Peters, a family medicine resident, about tips and best practices for managing money and postgraduate student loan debt as a young professional.</description>
+            <enclosure
+                length="9393384"
+                type="audio/mpeg"
+                url="https://files.consumerfinance.gov/f/media/cfpb_financial-intuition-podcast_managing-your-student-loans_episode-3.mp3"
+            />
+            <pubDate>Tue, 15 Dec 2020 16:00:00 EDT</pubDate>
+            <itunes:duration>1547</itunes:duration>
+            <itunes:explicit>false</itunes:explicit>
+        </item>
+        <item>
+            <title>Managing Your Student Loans, Part 2: Discussing the Veterans Benefits Administration’s (VBA) GI Bill® Comparison Tool and CFPB’s Your Financial Path to Graduation Tool</title>
+            <link>https://www.consumerfinance.gov/practitioner-resources/students/financial-intuition/managing-your-student-loans-part-2/</link>
+            <itunes:episodeType>full</itunes:episodeType>
+            <itunes:episode>5</itunes:episode>
+            <description>During the second episode in this series, we met with Charmain Bogue, Executive Director of Education Service for the Veterans Benefits Administration, to discuss tools to help veterans, servicemembers, and their qualified family members with making decisions about financing their education and selecting an educational institution to attend.</description>
+            <enclosure
+                length="8634384"
+                type="audio/mpeg"
+                url="https://files.consumerfinance.gov/f/media/cfpb_financial-intuition-podcast_managing-your-student-loans_episode-2.mp3"
+            />
+            <pubDate>Tue, 15 Dec 2020 16:00:00 EDT</pubDate>
```

Comment:
Let's add one second to each pubDate as we did previously, in case people's podcast apps sort by that.

```suggestion
            <pubDate>Tue, 15 Dec 2020 16:00:02 EDT</pubDate>
            <itunes:duration>1547</itunes:duration>
            <itunes:explicit>false</itunes:explicit>
        </item>
        <item>
            <title>Managing Your Student Loans, Part 2: Discussing the Veterans Benefits Administration’s (VBA) GI Bill® Comparison Tool and CFPB’s Your Financial Path to Graduation Tool</title>
            <link>https://www.consumerfinance.gov/practitioner-resources/students/financial-intuition/managing-your-student-loans-part-2/</link>
            <itunes:episodeType>full</itunes:episodeType>
            <itunes:episode>5</itunes:episode>
            <description>During the second episode in this series, we met with Charmain Bogue, Executive Director of Education Service for the Veterans Benefits Administration, to discuss tools to help veterans, servicemembers, and their qualified family members with making decisions about financing their education and selecting an educational institution to attend.</description>
            <enclosure
                length="8634384"
                type="audio/mpeg"
                url="https://files.consumerfinance.gov/f/media/cfpb_financial-intuition-podcast_managing-your-student-loans_episode-2.mp3"
            />
            <pubDate>Tue, 15 Dec 2020 16:00:01 EDT</pubDate>
```